pkg/catalog/versions/k8s.go

```diff
@@ -50,6 +50,12 @@ var kubernetesVersions = map[string]KubernetesVersions{
 		Etcd:       "3.5.12",
 		CoreDNS:    "current",
 	},
+	"1.31.0": {
+		Kubernetes: "1.31.0",
+		Pause: "3.10",
+		Etcd: "3.5.15",
```

Comment:
Should this be `3.5.12` instead? Refer https://github.com/oracle-cne/ock/pull/23/files#diff-9a0aa0c2d4c623b94650f1495905e7af6142275086c63ab2d62f345ceefb7a28R31

Reply:
Same as with Kubernetes, except in this case the version is completely irrelevant.  It's only here because we need a value to put in the config files to pass validation.